packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -440,8 +441,6 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
         ref={ this._resolveRef('_splitButtonContainer') }
       >
         <span
-          aria-hidden={ true }
-          // TODO: THIS SHOULD BE REMOVED!
           style={ { 'display': 'flex' } }
         >
```

Comment:
Does this span even need to be here anymore?

Reply:
Yes. the style would expand our buttons to fit the height set by the div container.

Reply:
Take a look at how this renders, I don't think it's needed from the styling on the button(s)

Reply:
I've investigated this and I don't think either solution would work and it might be better to stick with the container.

The purpose of the span and display:flex is used to make the menu button the same height as the primary button.

I've investigated the solutions you made and:

1) If we were to remove the span our menu button would have a different size as the primary button. 

To rectify situation I had to find any height value specified by the user either as a specific style if it exists and if not grab it from the root class name. However root is of type "any" and while I think I can cast it to find a height attribute, I feel that it's more involved than necessary, it could also be problematic if a custom class styles is also given with height values. I guess we can solve all of this by introducing a height prop that we would explicitly use, but then we'll have backward compatibility problems.

2) If I were to add the display:flex; to the div that's the span's container, that would solve the height problem, however that would cause the div and the container to fill in the size of its container. 

This is fine by itself, but the problem then would be our divider which relies on absolute positioning will be sent to the far right of the container. We can fix that by setting the divider to use relative positioning, however when we do that we have a new problem, we can't take advantage of the top and bottom css properties that allows the divider to have the height it has now. To solve this, we can figure out what the height of the button is and make the necessary adjustments to the divider, but now we have to get the height, which goes back to the first problem.

So with all of that considered, unless there's another CSS technique we can use, I think getting rid of the span will cause more problems than just keeping the span there.

Reply:
Okay, thanks for the investigation! Yeah it sounds like just leave the span there